packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionHistoryTable.tsx

```diff
@@ -221,6 +251,9 @@ export const TransactionHistoryTable = ({
                 return null
               }
 
+              const bgColorSettled = isEvenRow ? 'bg-cyan' : 'bg-white'
```

Comment:
the new design won't have alternative row colours, might as well remove the condition to simplify it?
up to you